src/main/resources/templates/fragments/_bookShower.html

```diff
@@ -4,11 +4,14 @@ <h2 th:case="null">Book Catalogue is Empty!</h2>
 
         <div class="mdl-card mdl-shadow--2dp mdl-card--horizontal" th:each="book : ${books}" style="width:100%">
             <div class="mdl-card__media">
-                <img class = "card-image" th:src="${book.picture}" alt="img">
+                <img class = "card-image" th:src="${!book.picture_url.isEmpty()} ? ${book.picture_url} : 'https://via.placeholder.com/155x200'" alt="img">
```

Comment:
This is the old placeholder image which just says dimensions, seems like we forgot to push the updated version of this file with the 'book cover not found' specific message which is more user friendly.